README.md

```diff
@@ -13,79 +13,228 @@
+    // specific errors that can happen during buildpack execution. This error type should
+    // only contain error specific to this buildpack, such as CouldNotExecuteMaven or
+    // InvalidGemfileLock. This example buildpack uses GenericError which means this buildpack
     // does not specify any errors.
     //
-    // More generic errors that happen during buildpack execution such as I/O errors while 
+    // More generic errors that happen during buildpack execution such as I/O errors while
     // writing CNB TOML files are handled by libcnb.rs itself.
     type Error = GenericError;
 
-    // This method will be called when the CNB lifecycle calls detect. Use the DetectContext to 
-    // access CNB data such as the stack this buildpack is currently executed on, the app 
-    // directory and similar things. When using libcnb.rs, you never have to read environment 
+    // This method will be called when the CNB lifecycle calls detect. Use the DetectContext to
+    // access CNB data such as the stack this buildpack is currently executed on, the app
+    // directory and similar things. When using libcnb.rs, you never have to read environment
     // variables or read/write files to disk to interact with the CNB lifecycle.
     //
-    // One example of this is the return type of this method. DetectResult encapsulates the 
-    // required exit code as well as the data written to the build plan. libcnb.rs will, 
-    // according to the returned value, handle both writing the build plan and exiting with 
+    // One example of this is the return type of this method. DetectResult encapsulates the
+    // required exit code as well as the data written to the build plan. libcnb.rs will,
+    // according to the returned value, handle both writing the build plan and exiting with
     // the correct status code for you.
-    fn detect(&self, context: DetectContext<Self>) -> libcnb::Result<DetectResult, Self::Error> {
+    fn detect(&self, _context: DetectContext<Self>) -> libcnb::Result<DetectResult, Self::Error> {
         Ok(DetectResultBuilder::pass().build())
     }
 
-    // Similar to detect, this method will be called when the CNB lifecycle executes the 
+    // Similar to detect, this method will be called when the CNB lifecycle executes the
     // build phase.
     fn build(&self, context: BuildContext<Self>) -> libcnb::Result<BuildResult, Self::Error> {
         println!("Hello World!");
         println!("Build runs on stack {}!", context.stack_id);
-        Ok(BuildResultBuilder::new().build())
+
+        Ok(BuildResultBuilder::new()
+            .launch(Launch::new().process(Process::new(
+                process_type!("web"),
+                "echo",
+                vec!["Hello World!"],
+                false,
+                true,
+            )))
+            .build())
     }
 }
 
 // Implements the main function and wires up the framework for the given buildpack.
 buildpack_main!(HelloWorldBuildpack);
+
```

Comment:
(Means when using the "copy" button in the rendered markdown, the resultant text doesn't have two trailing empty lines)

```suggestion
```